cli/install/template.go

```diff
@@ -188,20 +188,6 @@ spec:
         - "tap"
         - "-log-level={{.ControllerLogLevel}}"
         - "-logtostderr=true"
-      - name: telemetry
```

Comment:
you can also remove references to the deprecated `controller` prometheus job:
https://github.com/runconduit/conduit/blob/2f5b5ea5f20152ea0ab556fcdaf7fc4d8f0d94a3/cli/install/template.go#L345-L359

you can also remove `telemetry` from prometheus in our docker-compose environment:
https://github.com/runconduit/conduit/blob/2f5b5ea5f20152ea0ab556fcdaf7fc4d8f0d94a3/.prometheus.dev.yml#L15